assets/constants.ts

```diff
@@ -5438,7 +5438,7 @@ export = {
   "GCS_RESOURCE_BUCKET_NAME": "None-resources",
 
   // Used to disable account removal until it is fully implemented.
-  "ENABLE_ACCOUNT_DELETION": false,
+  "ENABLE_ACCOUNT_DELETION": true,
```

Comment:
Perhaps this flag flip should go in a separate PR that is only merged once the entirety of Wipeout is confirmed to work correctly.

Alternatively, you can make it a config property so that it can be adjusted on the fly.

Reply:
Done, removed the change. I decided not to make it into config property since it would require to make a HTTP request since we need the constant in the frontend.